### PR TITLE
Added a "kits" folder, with an "arm" subfolder.

### PR DESCRIPTION
**Pull request by iamtesch**
- Initial example: simple gravity compensation
- Also added container for arm information.

### CODE REVIEW

File: kits/arm/gravity_compensation.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  // robot_model object and mass vector.
+  std::unique_ptr<hebi::ArmContainer> arm = hebi::ArmContainer::create3Dof();
+  if (!arm)
+    return -1;
+
+  hebi::GroupCommand cmd(arm->getGroup().size());
+  Eigen::Vector3d gravity(0, 0, -1);
+ 
+  // Respond to every feedback packet with an effort command to cancel the
+  // force to to gravity at this pose.
```
Comment (rLinks234) [inline kits/arm/gravity_compensation.cpp]: comment typo!

File: util/grav_comp.hpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const Eigen::VectorXd& masses,
+  const Eigen::VectorXd& angles,
+  const Eigen::Vector3d& gravity)
+{
+  // Normalize gravity vector (to 1g, or 9.8 m/s^2)
+  Eigen::Vector3d normed_gravity = gravity;
+  normed_gravity /= normed_gravity.norm();
+  normed_gravity *= 9.81;
+
+  int num_dof = model.getDoFCount();
```
Comment (rLinks234) [inline util/grav_comp.hpp]: size_t

File: util/grav_comp.hpp [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  int num_frames = model.getFrameCount(HebiFrameTypeCenterOfMass);
```
Comment (rLinks234) [inline util/grav_comp.hpp]: size_t

File: util/grav_comp.hpp [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  // Get torque for each module
+  // comp_torque = J' * wrench_vector
+  // (for each frame, sum this quantity)
+  Eigen::VectorXd comp_torque(num_dof);
+  comp_torque.setZero();
+
+  // Wrench vector
+  Eigen::VectorXd wrench_vec(6); // For a single frame; this is (Fx/y/z, tau x/y/z)
+  wrench_vec.setZero();
+  for (int i = 0; i < num_frames; ++i)
```
Comment (rLinks234) [inline util/grav_comp.hpp]: size_t

File: util/grav_comp.hpp [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  {
+    // Set translational part
+    for (int j = 0; j < 3; ++j)
```
Comment (rLinks234) [inline util/grav_comp.hpp]: size_t

Comment (rLinks234) [reply util/grav_comp.hpp]: (sorry)

Comment (iamtesch) [reply util/grav_comp.hpp]: :) Good catches.  It was old code I was copy/pasting...

**Pull request merged**